### DISCUSSION TITLE
New hosting release?

### DISCUSSION THREAD

**Discussion by DenjixPochita** [plausible/analytics#1944] [Self-Hosted Support] [answered]
After seeing this, https://github.com/plausible/hosting/pull/45#issuecomment-1114550553. I thought we are getting a new release, but there hasn't been one yet. Not asking for any ETA, just confirming that, are we getting a new release?
Thank You!

**Comment by metmarkosaric** [comment, marked as answer]
Thanks @DenjixPochita! Self-hosted version is not updated as frequently as the hosted version which changes daily so latest features won’t be immediately available. Think of it as a long term release. We stack all the new features and fixes that have been battle tested on the hosted version and release a new self-hosted version twice a year or so. The last release was on January 26th so the next one should be there at some point later this year. We don't have an ETA.

**Reply by DenjixPochita** [reply]
Thanks got it!
